project/cmake/addons/addons/adsp.basic/adsp.basic.txt

```diff
@@ -1 +1 @@
-adsp.basic https://github.com/AchimTuran/adsp.basic master
+adsp.basic https://github.com/AchimTuran/adsp.basic fix_win32_cmake
```

Comment:
What is the correct whitespace betwern the "url" and "origin/branch"?

I tried this: adsp.basic https://github.com/AchimTuran/adsp.basic origin/fix_win32_cmake
and with "tools\buildsteps\win32\make-addons.bat".

This is the error:
```
CMake Error at CMakeLists.txt:173 (message):
ERROR downloading
https://github.com/AchimTuran/adsp.basic/archive/origin/fix_win32_cmake.tar.gz

- status: 22;"HTTP response code said error" log: About to connect() to github.com port 443 (#0)

Trying 192.30.252.131... connected

Connected to github.com (192.30.252.131) port 443 (#0)

libcurl is now using a weak random seed!

SSLv3, TLS handshake, Client hello (1):
☺
-- Configuring incomplete, errors occurred!
See also "C:/xbmc-dev/kodi-15-adsp-alwin_neu/project/cmake/addons/build/CMakeFiles/CMakeOutput.log".
Failed to build addons See C:\xbmc-dev\kodi-15-adsp-alwin_neu\tools\buildsteps\win32\..\..\..\project\cmake\make-addons.error for more details
´´´